convene-web/app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class InvitationsController < ApplicationController
+
+  # TODO: fix create action to create invitation data so spec test passes
+  def create
+    @space_invitation = current_space.invitations.new(invitation_params)
```

Comment:
Remove instance variable and rename variable to reference controller name for clarity